genie-web/src/test/java/com/netflix/genie/web/configs/PropertiesConfigIntegrationTest.java

```diff
@@ -40,7 +41,10 @@
 @Category(IntegrationTest.class)
 @RunWith(SpringRunner.class)
 @TestPropertySource(locations = "classpath:/PropertiesConfigIntegrationTest.properties")
-@SpringBootTest()
+@SpringBootTest(
+    classes = GenieTestApp.class,
+    properties = {"genie.grpc.server.enabled = false"}
```

Comment:
Why not put this in the YAML?

Reply:
Cause right now the yaml defaults to having it on. I'm thinking of turning it off by default and having only the tests that need it opt in